doozerlib/rhcos.py

```diff
@@ -354,7 +354,13 @@ def get_package_build_objects(self) -> Dict[str, Dict]:
         aggregate: Dict[str, Dict] = dict()
         with self.runtime.pooled_koji_client_session() as koji_api:
             for nvra in self.get_rpm_nvras():
-                rpm_def = koji_api.getRPM(nvra, strict=True)
+                try:
+                    rpm_def = koji_api.getRPM(nvra, strict=True)
+                except koji.GenericError as e:
+                    if self.runtime.group_config.rhcos.allow_missing_brew_rpms:
```

Comment:
Let's be more specific:
```suggestion
                    if self.runtime.group_config.rhcos.allow_missing_brew_rpms and "No such rpm" in str(e):
```